### ISSUE TITLE
Prevent clicks when dropping scrollbar handle

### ISSUE THREAD

**Issue by tryggvigy** [KingSora/OverlayScrollbars#251]
**Describe the bug**
Wasn't sure if I should raise this as a bug report or a feature request.

When the overlay scrollbar handle is dropped, a click event also fires.
I think this is different from native scrollbars (I believe) and is undesired in most cases I think.

Let's say I have a list of items, like a playlist on Spotify. You can select tracks in the playlist by clicking on them. If I now drag the overlay scrollbar and release it either on top of `body` or on top of the handle itself, a click event is also fired. The playlist is built to clear selection of it's tracks when a user clicks outside of the playlist. This is a common pattern and often used to close modals when a user clicks outside:
```js
// Inside some component
document.body.addEventListener(('click') => {
  // did click happen outside me?
  // if yes, do some state clean-up (like closing modal or clearing selection)
})
```

In our case I worked around this by disabling pointer-events on `body` and the handle while it's being dragged.

```scss
// Prevent clicks on body and scrollbar handle when dragging and dropping the handle.
// We for example don't want to clear the selected tracks in a playlist when
// the scrollbar handle is dropped. The .os-dragging class is only applied
// while the overlay scrollbars are being dragged with the mouse.
.os-dragging,
.os-dragging .os-scrollbar-handle {
  pointer-events: none !important;
}
```

Happy to fill in a js-fiddle and the rest of the steps in the template later if this is not clear enough. I don't have time right now

---

**To Reproduce**

**Expected behavior**

**Examples**
Please create a small example of the bug. 
To do this you can use online platforms like [JSFiddle](https://jsfiddle.net/), [CodeSandbox](https://codesandbox.io/) or [StackBlitz](https://stackblitz.com/). You can also create a separate Github repository which I can clone.

**Environment**
 - Used Operating System(s):
 - Used Browser(s) (with version):

**Additional context**

**Comment by KingSora** [comment]
Good day!

Thanks for the report, and yeah, you're right. I made a POC here: https://jsfiddle.net/Lmd73uw5/1/ which is solved in JS, because I'm not sure about the CSS Solution you fixed it with, you can decide which solution is better / more stable.

**Comment by tryggvigy** [comment]
Yeah that makes sense. Your JS solution more directly solves the problem, also quite neat and clever!
I guess we can consider this issue resolved since people facing this issue can follow this advice.

Question is, do you think this should be handled internally by OverlayScrollbars to more closely behave like native scrollbars or could there be edge-cases that make this not blanket apply to every user of the library by default?

**Comment by KingSora** [comment]
In this particular case I believe it is safe to implement it for every user, since users which used their native scrollbars before didn't have the `click` event before. So this solution doesn't take any functionality away which was there before.

I'll implement this enhancement in the next release, I'll leave this ticket open until then. Thanks again for the report!

**Comment by KingSora** [comment]
Implemented in [v1.13.0](https://github.com/KingSora/OverlayScrollbars/releases/tag/v1.13.0).

**Comment by tryggvigy** [comment]
Just tested this in 1.13.0, works great. Thanks!
